tool/gravity/cli/plan.go

```diff
@@ -82,18 +88,23 @@ func displayOperationPlan(localEnv, updateEnv, joinEnv *localenv.LocalEnvironmen
 	case ops.OperationInstall:
 		return displayInstallOperationPlan(op.Key(), format)
 	case ops.OperationExpand:
-		return displayExpandOperationPlan(joinEnv, op.Key(), format)
+		return displayExpandOperationPlan(environ, op.Key(), format)
 	case ops.OperationUpdate:
-		return displayUpdateOperationPlan(localEnv, updateEnv, op.Key(), format)
+		return displayUpdateOperationPlan(localEnv, environ, op.Key(), format)
 	case ops.OperationUpdateRuntimeEnviron:
-		return displayUpdateOperationPlan(localEnv, updateEnv, op.Key(), format)
+		return displayUpdateOperationPlan(localEnv, environ, op.Key(), format)
 	case ops.OperationUpdateConfig:
-		return displayUpdateOperationPlan(localEnv, updateEnv, op.Key(), format)
+		return displayUpdateOperationPlan(localEnv, environ, op.Key(), format)
 	case ops.OperationGarbageCollect:
 		return displayClusterOperationPlan(localEnv, op.Key(), format)
 	default:
 		return trace.BadParameter("unknown operation type %q", op.Type)
 	}
+	if err != nil && trace.IsNotFound(err) {
```

Comment:
What is this fallback for? Seems like the switch above covers all operation types?

Reply:
Looks like a backport gone slightly wrong - I'll fix that.